src/extensions/default/InlinePaddingEditor/PaddingUtils.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2013 - present Adobe Systems Incorporated. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+/**
+ *  Utilities regular expressions related to padding rule matching
+ *
+ */
+define(function (require, exports, module) {
+    "use strict";
+    /**
+     * Regular expression that matches the css rule for paddiing values after the
+     * colon is optional
+     * @const @type {RegExp}
+     */
+    var PADDING_REGEX = new RegExp('.*padding:.*');
```

Comment:
So "no" to `padding-top` and friends?

Reply:
Currently, yes quick edit UI is added for ``padding:`` only. 
I was referring to the recently merged PR for the border-radius. In that border-top-left-radius and friends are not set by using quick edit UI.
Do you want to add padding:top and others too (It would lead to change in UI as well) ?

Reply:
for now I would say let's just do `padding` and add the other properties incrementally in other issues.